base_view_inheritance_extension/models/ir_ui_view.py

```diff
@@ -125,7 +125,21 @@ def inheritance_handler_xpath(self, source, specs, inherit_id):
         target_node = self.locate_node(
             source, etree.Element(specs.tag, expr=specs.get('target'))
         )
-        target_node.append(node)
+
+        target_position = specs.get('target_position') or 'inside'
```

Comment:
pass `'inside'` as second parameter to `get`

Reply:
> target_position = specs.get('target_position', 'inside')

OK